flow/utils/evervigil/ever_vigil.go

```diff
@@ -0,0 +1,120 @@
+package evervigil
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/nikoksr/notify"
```

Comment:
This library doesn't seem to support emailing as a service, so why not skip it & use `github.com/slack-go/slack` directly?

Reply:
it does support e-mail, but switched to `slack-go` anyway